library/ssl_tls.c

```diff
@@ -4826,16 +4914,23 @@ static void ssl_bitmask_set( unsigned char *mask, size_t offset, size_t len )
 static int ssl_bitmask_check( unsigned char *mask, size_t len )
 {
     size_t i;
+    volatile unsigned char *mask_dup = mask;
+    volatile size_t len_dup = len;
 
     for( i = 0; i < len / 8; i++ )
         if( mask[i] != 0xFF )
-            return( -1 );
+            return( 0x75555555 );
```

Comment:
Please do not use magic numbers. Maybe Hamming distances variables should be defined as a global.